packages/react-ai/src/types.ts

```diff
@@ -41,4 +41,8 @@ export interface SendMesageParameters {
 
 export type SendMessage = (input: SendMesageParameters) => void;
 
+type V6Client<T extends Record<any, any>> = Pick<
```

Comment:
Is the any type required?

Reply:
Prefer not replicating a narrowed `V6Client` interface to prevent it leaking out of this file causing conflicts with the existing interface. Can we narrow the conditional inference of `Schema`?